src/libertem/udf/holography.py

```diff
@@ -139,7 +138,7 @@ def get_result_buffers(self):
         else:
             dtype = np.complex128
         return {
-            "wave": self.buffer(kind="nav", dtype=dtype, extra_shape=extra_shape)
+            "wave": self.buffer(kind="nav", dtype=dtype, extra_shape=extra_shape, where='device')
```

Comment:
The final result on the leader node will be allocated in RAM even if where='device' since the leader node may not even have a GPU. Nevertheless, I've switched it to a version that exports right away, with a FIXME to check what is better.